contracts/modules/STO/USDTieredSTO.sol

```diff
@@ -262,7 +262,7 @@ contract USDTieredSTO is USDTieredSTOStorage, ISTO, ReentrancyGuard {
         }
         usdTokens = _usdTokens;
         for(i = 0; i < _usdTokens.length; i++) {
-            require(_usdTokens[i] != address(0), "Invalid USD token");
+            require(_usdTokens[i] != address(0) && _usdTokens[i] != address(0), "Invalid USD token");
```

Comment:
isn't `address(0)` need to replace with `polyToken`.
`require(_usdTokens[i] != address(0) && _usdTokens[i] != address(polyToken), "Invalid USD token");`

Reply:
Oh yes. It was a typo.